patch-libxl-add-pcidevs-to-stubdomain-earlier.patch

```diff
@@ -0,0 +1,178 @@
+From f0cd65d9fd11b51bdb2adc6a15cf78799083a7ae Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
+ <marmarek@invisiblethingslab.com>
+Date: Thu, 30 Dec 2021 21:12:00 +0100
+Subject: [PATCH] libxl: add pcidevs to stubdomain earlier
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+Organization: Invisible Things Lab
+Cc: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>
```

Comment:
Does this header need to be encoded the way `From: ` was?

Reply:
Doesn't matter